__tests__/api/app.test.js

```diff
@@ -12,35 +13,55 @@ beforeEach(() => {
   return seed(seedTestData);
 });
 
-describe('Testing app', () => {
-  describe('Endpoint /api/topics', () => {
-    describe('Method GET', () => {
-      test('GET: 200 status', () => {
-        return request(app).get('/api/topics').expect(200);
+describe("Testing app", () => {
+  describe("Endpoint /api/topics", () => {
+    describe("Method GET", () => {
+      test("GET: 200 status", () => {
+        return request(app).get("/api/topics").expect(200);
       });
 
-      test('GET: 200 status with all topics data and properties', () => {
+      test("GET: 200 status with all topics data and properties", () => {
         return request(app)
-          .get('/api/topics')
+          .get("/api/topics")
           .expect(200)
           .then(({ body }) => {
             const topics = body;
             expect(Array.isArray(topics)).toBe(true);
             expect(topics).toHaveLength(3);
 
             topics.forEach((item) => {
-              expect(item).toHaveProperty('slug', expect.any(String));
-              expect(item).toHaveProperty('description', expect.any(String));
+              expect(item).toHaveProperty("slug", expect.any(String));
+              expect(item).toHaveProperty("description", expect.any(String));
             });
```

Comment:
This is nice testing, I would recommend checking out _.toMatchObject_ in the jest docs as it might be useful here to clean it up slightly

Reply:
done